Files/Strings/en-US/Resources.resw

```diff
@@ -1884,4 +1884,7 @@
   <data name="DrivesWidgetMoreOptionsButton.AutomationProperties.Name" xml:space="preserve">
     <value>More drives options</value>
   </data>
+  <data name="SidebarFavorites" xml:space="preserve">
+    <value>Quick access</value>
```

Comment:
```suggestion
    <value>Favorites</value>
```

Reply:
submitted on commit: yaichenbaum rename the quick access section to favorites section